include/uapi/sound/sof/abi.h

```diff
@@ -59,4 +61,20 @@
 /* SOF ABI magic number "SOF\0". */
 #define SOF_ABI_MAGIC		0x00464F53
 
+#define SOF_MANIFEST_DATA_TYPE_NHLT 1
+
+struct sof_manifest_data {
+	__u32 type;
+	__u32 size;
```

Comment:
can we add descriptions for each of these fields? does size include the "header" part ie type+size or just size of data?

Reply:
I think they we're very self descriptive, but sure. size doesn't contain header, so that should be mentioned. I'm not sure what would be preferred way with size. For example nhlt has structs where size contains the header and structs where size doesnt contain the header (which is pretty annoying). I'm fine with both.

Reply:
added comments